custom_example_rules/validate_docstrings.py

```
@@ -0,0 +1,24 @@
+    description="Validate docstrings in the code.",
+    hooks=[DevRulesEvent.PRE_COMMIT],
+    ignore_defaults=True,
+)
+def check_docstrings(path: str = "src", fail_under: int = 98) -> tuple[bool, str]:
+    """Validate docstrings in the code.
+
+    Example interrogate response:
+        RESULT: PASSED (minimum: 80.0%, actual: 98.3%)
+
+    """
+    result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)